core/src/components/link/link.scss

```
@@ -1,4 +1,4 @@
-.sdds-link-component {
+.tds-link-component {
```

```suggestion
.sdds-link-component {
```
I think we need to keep this one sdds for a bit more time until we merge prefix-change branch back to the main branch.